src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -2119,7 +2119,8 @@ internal void GenerateAnonymousFunctionConversionError(BindingDiagnosticBag diag
 
             // If any parameter type of the lambda is an error type then suppress
             // further errors. We've already reported errors on the bad type.
-            if (anonymousFunction.HasExplicitlyTypedParameterList)
+            var hasExplicitlyTypedParameterList = anonymousFunction.HasExplicitlyTypedParameterList;
+            if (hasExplicitlyTypedParameterList)
```

Comment:
>hasExplicitlyTypedParameterList [](http://example.com/codeflow?start=16&length=31)

Consider inlining the value (that is, revert the change) since it's the only use.

Reply:
done.